docs/management/admin/response-actions-config.asciidoc

```diff
@@ -80,6 +81,67 @@ IMPORTANT: Do not create more than one CrowdStrike connector.
 This gives you visibility into CrowdStrike without needing to leave {elastic-sec}. You can perform supported endpoint response actions directly from alerts that a rule creates, by using the **Take action** menu in the alert details flyout.
 ====
 
+.**Set up Microsoft Defender for Endpoint response actions**
+[%collapsible]
+====
+// NOTE TO CONTRIBUTORS: These sections have very similar content. If you change anything 
+// in this section, apply the change to the other sections, too.
+
+. **Create API access information in Microsoft Azure.** You should create two new applications in your Azure domain and grant them the following minimum API permissions:
++
+--
+- For use with the Microsoft Defender for Endpoint Fleet integration policy: Permission to read alert data (`Windows Defender ATP: Alert.Read.All`).
+- For use with the Microsoft Defender for Endpoint connector: Permission to read machine information as well as isolate and release a machine (`Windows Defender ATP: Machine.Isolate and Machine.Read.All`).
+--
++
+Refer to the {integrations-docs}/microsoft_defender_endpoint[Microsoft Defender for Endpoint integration documentation] or https://learn.microsoft.com/en-us/defender-endpoint/api/exposed-apis-create-app-webapp[Microsoft's documentation] for details.
++
+After the applications are created, you should have the following 3 pieces of information available for each one:
++
+--
+- Client ID
+- Client secret
+- Tenant ID
+--
+
+. **Install the Microsoft Defender for Endpoint integration and {agent}.** Elastic's {integrations-docs}/microsoft_defender_endpoint[Microsoft Defender for Endpoint integration] collects and ingests logs into {elastic-sec}.
```

Comment:
I'm thinking we should also mention here that a user could also setup the "Microsoft M365 Defender" integration as an alternative (or additional) source for data... We tested this and it seems to work with bi-directional response actions. 

I don't think we should have an entire bullet section for it - perhaps just a callout or "note" to indicate we have support for it.

cc/ @raqueltabuyo , @caitlinbetz